api/proto/metrics/metrics.proto

```diff
@@ -429,6 +435,8 @@ message ZMetricProcess {
   uint64 vm_bytes = 11;
   uint64 rss_bytes = 12;
   float memory_percent = 13;
+  // We report the stack for !watched proceeses which have been running for a while
```

Comment:
The watched boolean is set by looking at the pids registered in /run/watchdog/pid/.

The idea is to avoid reporting stack traces for the watched processes since they are a lot likely to get stuck and the stack traces take up space in the message. (And the controller has a max size for the whole metrics message.)

Reply:
Any chance you can add the above text (or a portion of it) to the comment section?